packages/react/src/core/config.tsx

```diff
@@ -259,13 +259,20 @@ export function getFieldShapeClient(
   }
 
   if (isMediaFieldShape(fieldShape)) {
-    return R.omit(
-      {
-        ...fieldShape,
-        label: fieldShape.label ?? name,
-      },
-      ['$server', 'options' as any]
-    ) as FieldShapeClient & { $client: { fieldName: string } }
+    const base = {
+      ...fieldShape,
+      label: fieldShape.label ?? name,
+    } as any
+    const uploadOptions =
+      base.uploadOptions ??
+      (base.options ? base.options : undefined) ??
+      (base.mimeTypes ? { mimeTypes: base.mimeTypes } : undefined)
```

Comment:
This seems not correct. uploadOptions is defined as 

```tsx
export interface FileUploadOptionsProps {
  mimeTypes?: string[]
  maxSize?: number // Bytes (Default is 1024 * 1024 = 1 MB)
  /**
   * @description For now we accept only 1 file at a time (Collection constract accept only one key)
   */
  limit?: 1
  accept?: string
}
```

base.options in this case is always `undefined` and `base.mimeTypes` does not exists